src/Enable.Extensions.Queuing.RabbitMQ/Internal/RabbitMQQueueClient.cs

```diff
@@ -56,10 +56,7 @@ public RabbitMQQueueClient(ConnectionFactory connectionFactory, string queueName
                 autoDelete: false,
                 arguments: queueArguments);
 
-            // Here we are assuming a "worker queue", where multiple consumers are
-            // competing to draw from a single queue in order to spread workload
-            // across the consumers.
-            _channel.BasicQos(prefetchSize: 0, prefetchCount: 1, global: false);
```

Comment:
We may be able remove this line. I have not yet determined if the QoS applies to both the _pull API_ and _push API_, or just the _push API_. Without this line, the consumer connects with a `prefetchCount` of `0`, which as far as I know, means RabbitMQ may try to deliver an unlimited number of unacknowledged messages. If this indeed applies to the pull API, this would be undesirable and we will want to retain this line.